plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastoreConfig.java

```diff
@@ -37,6 +39,7 @@
     private Optional<String> awsAccessKey = Optional.empty();
     private Optional<String> awsSecretKey = Optional.empty();
     private Optional<String> awsCredentialsProvider = Optional.empty();
+    private Optional<File> awsCredentialsProviderConf = Optional.empty();
```

Comment:
I think it should be simply a string. It is up to custom credential provider what it is.